mailet/icalendar/src/test/java/org/apache/james/transport/mailets/ICALToJsonAttributeTest.java

```diff
@@ -304,6 +304,41 @@ public void serviceShouldAttachJsonForSeveralICALs() throws Exception {
+        testee.init(FakeMailetConfig.builder().build());
+
+        Calendar calendar = new CalendarBuilder().build(ClassLoader.getSystemResourceAsStream("ics/meeting.ics"));
+        Calendar calendar2 = new CalendarBuilder().build(ClassLoader.getSystemResourceAsStream("ics/meeting_without_uid.ics"));
+        ImmutableMap<String, Calendar> icals = ImmutableMap.<String, Calendar>builder()
+            .put("key", calendar)
+            .put("key2", calendar2)
+            .build();
+        MailAddress recipient = MailAddressFixture.OTHER_AT_JAMES;
+        Mail mail = FakeMail.builder()
+            .sender(SENDER)
+            .recipient(recipient)
+            .attribute(ICALToJsonAttribute.DEFAULT_SOURCE_ATTRIBUTE_NAME, icals)
+            .build();
+        testee.service(mail);
+
+        Map<String, byte[]> jsons = (Map<String, byte[]>) mail.getAttribute(ICALToJsonAttribute.DEFAULT_DESTINATION_ATTRIBUTE_NAME);
+        assertThat(jsons).hasSize(1);
+        List<String> actual = toSortedValueList(jsons);
+
+        assertThatJson(actual.get(0)).isEqualTo("{\n" +
+            "\t\"ical\": \"BEGIN:VCALENDAR\\r\\nPRODID:-//Aliasource Groupe LINAGORA//OBM Calendar 3.2.1-rc2//FR\\r\\nCALSCALE:GREGORIAN\\r\\nX-OBM-TIME:1483703436\\r\\nVERSION:2.0\\r\\nMETHOD:REQUEST\\r\\nBEGIN:VEVENT\\r\\nCREATED:20170106T115035Z\\r\\nLAST-MODIFIED:20170106T115036Z\\r\\nDTSTAMP:20170106T115036Z\\r\\nDTSTART:20170111T090000Z\\r\\nDURATION:PT1H30M\\r\\nTRANSP:OPAQUE\\r\\nSEQUENCE:0\\r\\nSUMMARY:Sprint planning #23\\r\\nDESCRIPTION:\\r\\nCLASS:PUBLIC\\r\\nPRIORITY:5\\r\\nORGANIZER;X-OBM-ID=128;CN=Raphael OUAZANA:MAILTO:ouazana@linagora.com\\r\\nX-OBM-DOMAIN:linagora.com\\r\\nX-OBM-DOMAIN-UUID:02874f7c-d10e-102f-acda-0015176f7922\\r\\nLOCATION:Hangout\\r\\nCATEGORIES:\\r\\nX-OBM-COLOR:\\r\\nUID:f1514f44bf39311568d640727cff54e819573448d09d2e5677987ff29caa01a9e047feb2aab16e43439a608f28671ab7c10e754ce92be513f8e04ae9ff15e65a9819cf285a6962bc\\r\\nATTENDEE;CUTYPE=INDIVIDUAL;RSVP=TRUE;CN=Matthieu EXT_BAECHLER;PARTSTAT=NEEDS-ACTION;X-OBM-ID=302:MAILTO:baechler@linagora.com\\r\\nATTENDEE;CUTYPE=INDIVIDUAL;RSVP=TRUE;CN=Laura ROYET;PARTSTAT=NEEDS-ACTION;X-OBM-ID=723:MAILTO:royet@linagora.com\\r\\nATTENDEE;CUTYPE=INDIVIDUAL;RSVP=TRUE;CN=Raphael OUAZANA;PARTSTAT=ACCEPTED;X-OBM-ID=128:MAILTO:ouazana@linagora.com\\r\\nATTENDEE;CUTYPE=INDIVIDUAL;RSVP=TRUE;CN=Luc DUZAN;PARTSTAT=NEEDS-ACTION;X-OBM-ID=715:MAILTO:duzan@linagora.com\\r\\nATTENDEE;CUTYPE=RESOURCE;CN=Salle de reunion Lyon;PARTSTAT=ACCEPTED;X-OBM-ID=66:MAILTO:noreply@linagora.com\\r\\nATTENDEE;CUTYPE=INDIVIDUAL;RSVP=TRUE;CN=Antoine DUPRAT;PARTSTAT=NEEDS-ACTION;X-OBM-ID=453:MAILTO:duprat@linagora.com\\r\\nATTENDEE;CUTYPE=INDIVIDUAL;RSVP=TRUE;CN=\\\"Benoît TELLIER\\\";PARTSTAT=NEEDS-ACTION;X-OBM-ID=623:MAILTO:tellier@linagora.com\\r\\nATTENDEE;CUTYPE=INDIVIDUAL;RSVP=TRUE;CN=Quynh Quynh N NGUYEN;PARTSTAT=NEEDS-ACTION;X-OBM-ID=769:MAILTO:nguyen@linagora.com\\r\\nEND:VEVENT\\r\\nEND:VCALENDAR\\r\\n\",\n" +
```

Comment:
I improved readability of these JSON

However, as explained to Matthieu, I can not use the content of INPUT files : 
 - Content differ due to the output of ICAL serialization with ICAL4J
 - Needs to be escaped